### DISCUSSION TITLE
Julia Examples

### DISCUSSION THREAD

**Discussion by jjallaire** [quarto-dev/quarto-cli#1261] [Show and Tell]
Hi everyone, I am giving a talk on Quarto later on this summer at [JuliaCon](https://juliacon.org/2022/). I am hoping to assemble lots of high-quality examples of using Julia to create website, books, blogs, presentations, etc. Would very much appreciate if people could post examples they are aware of to this thread!

**Comment by mroavi** [comment]
Hi @jjallaire. Nice! I'm actually planning on giving a talk at JuliaCon too and I will be using Quarto for the slides!

The name of my talk is: [JunctionTrees](https://github.com/mroavi/JunctionTrees.jl): Bayesian inference in discrete graphical models (I might still change a few words if they allow me)

**Reply by jjallaire** [reply]
Awesome! Is there a link to slides and/or source code yet?

**Reply by mroavi** [reply]
I have been working locally so far. But I can put it in a git repo and share it with you. It is still work-in-progress.

**Reply by mroavi** [reply]
Here you go: https://github.com/mroavi/juliacon-2022

**Reply by mroavi** [reply]
@jjallaire Could you teach me how to host my quarto-reaveal.js presentation as you did here: https://quarto.org/docs/presentations/revealjs/demo/#/title-slide
I uploaded all the generated files to the root of my main branch and I set the GitHub Pages site to be built from the main branch. However, the presentation doesn't work still (which you can see by clicking the link above). I noticed the `self-contained` option. Do I need to do something with that?

**Reply by jjallaire** [reply]
It looks like this site was correctly published: https://mroavi.github.io/juliacon-2022/

**Reply by mroavi** [reply]
@jjallaire But it doesn't display as a reveal.js presentation in *presentation mode*, like yours here: https://quarto.org/docs/presentations/revealjs/demo/#/title-slide.

**Reply by jjallaire** [reply]
The index.html file in your repo is not rendered as revealjs presentation (it's just a normal html document). You need to re-render the qmd.

**Reply by mroavi** [reply]
@jjallaire Ahh my bad! You are right. I was rendering to `html`. I rendered it to `revealjs` and pushed the resulting index.html file but it still doesn't display correctly. Any idea why?

**Reply by mroavi** [reply]
Ohh wait. I think I didn't push the index_files folder. Let me retry.

**Reply by mroavi** [reply]
@jjallaire it worked 🎉 Thanks!!

**Comment by jjallaire** [comment]
Just noticed that @pat-alt has his blog running on Quarto: https://www.paltmeyer.com/blog/

**Reply by pat-alt** [reply]
I do and I'm loving it! One of the latest posts is actually about how much I've enjoyed using Julia + Quarto so far 😆

**Reply by pat-alt** [reply]
Also prepared my JuliaCon talks using Quarto (final polishing to do tomorrow, but basically done):

- [CounterfactualExplanations.jl](https://github.com/pat-alt/CounterfactualExplanations.jl/blob/main/dev/resources/juliacon22/presentation.qmd) (main talk)
- [LaplaceRedux.jl](https://github.com/pat-alt/LaplaceRedux.jl/blob/main/dev/resources/juliacon22/presentation.qmd) (lightening talk)

**Reply by mroavi** [reply]
Your second package looks really interesting! I'm gonna try it out. By the way, in case this might be of interest, I posted a two-part series intro on the Laplace Approximation [here](https://forem.julialang.org/mroavi/the-laplace-approximation-part-1-2cob) in the Julia forem.

**Reply by pat-alt** [reply]
Thanks @mroavi , I'm quite excited about the topic! Actually came across your post on forem a while ago, nice work 👍🏽

**Comment by jeremiahpslewis** [comment]
Here's a self-contained quarto repo with Julia code snippets and GitHub actions wired up to auto-generate a site: https://github.com/jeremiahpslewis/quarto-reproducible-gh-actions

Rendered site link: https://jeremiahpslewis.github.io/quarto-reproducible-gh-actions/julia-supporting-material.html

**Comment by Gabrielforest** [comment]
Hi! I have got an example https://github.com/Gabrielforest/kaggle_competitions/tree/main/digit_recognizer and maybe some of my connections from Linkedin are using Julia with Quarto package. Can I post your question there?

**Reply by jjallaire** [reply]
Thank you! Is there a place where you've published the rendered document that I could also link to?

Yes, by all means post the question on LinkdedIn!

**Reply by Gabrielforest** [reply]
I just posted it https://gabrielforest.github.io/julia_qmd/ 
Also shared your question here https://www.linkedin.com/feed/update/urn:li:activity:6948612646929592320/

**Comment by pat-alt** [comment]
Also a visual that might be of interest below (generated using `Javis.jl` borrowing heavily from this [tutorial](https://juliaanimators.github.io/Javis.jl/stable/tutorials/tutorial_7/)). Feel free to recycle or tweak (code can be found [here](https://gist.github.com/pat-alt/eeb0608437d7dd11355b1437b9f0fe62)).

![](https://raw.githubusercontent.com/pat-alt/blog/main/docs/posts/julia-and-quarto-a-match-made-in-heaven/www/intro.gif)

**Comment by AlissonRP** [comment]
Hi @jjallaire, I have this [repo](https://github.com/AlissonRP/gh_langs)  that I used Julia + Quarto, but at the moment the analysis is in Portuguese, but soon I will write in English too
